api/proto/teleport/legacy/types/types.proto

```diff
@@ -4412,7 +4412,11 @@ message LockTarget {
 
   // Node specifies the UUID of a Teleport node.
   // A matching node is also prevented from heartbeating to the auth server.
-  string Node = 4 [(gogoproto.jsontag) = "node,omitempty"];
+  // DEPRECATED: use ServerID instead.
```

Comment:
Up to you if you want to leave this in, `deprecated = true` already adds a deprecation doc comment to the field (but without the custom message).